test/optimizer/applyDCEGraphRemovals-output.js

```diff
@@ -1,8 +1,8 @@
 var name;
 
 var wasmImports = {
- "save1": 1,
- "save2": 2
+ save1: 1,
+ save2: 2
```

Comment:
Do we never have export names that need quoting for other reasons? For example, a name with a space in it, like this:

```cpp
__attribute__((import_module("module"),
               import_name("spaces in between"))) void func();
```

Reply:
I'm not sure that can work in our current code base, with or without this change.

If we want to support that kind of thing we should add a test for it.

Trying out the above today is currently not possible... even if I write a library js file like this:

```
mergeInto(LibraryManager.library, {
  "spaces in between": function() {
    console.log('XXX');
    return 0;
  },
});
```

Results in the following error at runtime:

```
$ node a.out.js 
/usr/local/google/home/sbc/dev/wasm/emscripten/a.out.js:1076
  function _spaces in between() {
                   ^^

SyntaxError: Unexpected token 'in'
```

So... this change isn't breaking anything that isn't already broken.